handler/src/main/java/io/netty/handler/pcap/packet/UDPPacket.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.pcap.packet;
+
+import io.netty.buffer.ByteBuf;
+
+public final class UDPPacket {
+
+    /**
+     * Create UDP Packet
+     *
+     * @param byteBuf ByteBuf where Packet data will be set
+     * @param payload Payload of this Packet
+     * @param srcPort Source Port
+     * @param dstPort Destination Port
+     */
+    public static ByteBuf createPacket(ByteBuf byteBuf, ByteBuf payload, int srcPort, int dstPort) {
```

Comment:
nit:  I think the method name is a bit misleading as we not really create a packet but we write it to the given `ByteBuf`. So consider to rename the method and use `void` as the return method to make it clear it not creates a new `ByteBuf`. 

Same comment is true for all the other places in the code with the same pattern

Reply:
Done